voice/guides/cnamService.md

```diff
@@ -0,0 +1,44 @@
+# CNAM per DIP API setup instructions
+
+## Overview
+Bandwidth's Caller Name (CNAM) service is a low-cost per-query service that provides the Caller Name on VoIP calls in the US and Canada.
+With this service, the CNAM database is queried using a simple HTTPS GET Request.
+Once the request is made, the calling name is returned to the customer application or dial plan.
+The CNAM data is managed by the phone company who provides the phone numbers to its customers.
+Bandwidth can not guarantee the validity of the information returned by the managing phone company.
+
+To utilize the CNAM lookup API, you will need to be setup with an account by your Account Manager.
+Please open a [support ticket](https://support.bandwidth.com/hc/en-us/requests/new) to enable this feature for your account.
+
+## CNAM setup instructions via HTTPS
+The simplest method of accessing the CNAM service is via an HTTPS GET request.
```

Comment:
are there other methods for accessing this specific service?

Reply:
Changed the way it was written. Maybe it makes a little more sense now?
There are other two ways (non-api) linked at the end of the doc.